phpmyfaq/src/phpMyFAQ/Controller/Administration/CategoryController.php

```diff
@@ -59,8 +59,9 @@ public function index(Request $request): Response
 
         $session = $this->container->get('session');
         $categoryOrder = $this->container->get('phpmyfaq.category.order');
-        $orderedCategories = $categoryOrder->getAllCategories();
-        $categoryTree = $categoryOrder->getCategoryTree($orderedCategories);
+        $orderedCategories = $categoryOrder->getAllCategories(); 
```

Comment:
This method returns the category ID as "category_id":

https://github.com/thorsten/phpMyFAQ/blob/main/phpmyfaq/src/phpMyFAQ/Category/Order.php#L170